file/cache.lisp

```diff
@@ -41,27 +52,52 @@
 (defmethod output-stream-p ((stream file-cache-stream))
   (member (direction stream) '(:output :io)))
 
+(defgeneric fs-read-block (stream block-n)
+  (:documentation "Read block-n from file"))
+
+(defgeneric fs-write-block (stream)
+  (:documentation "Write block-n to file"))
+
+;; TODO: Give error when out of space
+(defun make-block (stream)
+  (make-array (block-length stream)
+              :element-type '(unsigned-byte 8)
+              :area :wired))
```

Comment:
Should not be allocated in the wired area if they're going to exist indefinitely.